app/javascript/controllers/agents_statistics_chart.js

```diff
@@ -0,0 +1,63 @@
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
this need to be a stimulus controller,  see what we did with other graphs/chart in the summary page